http/vibe/http/server.d

```diff
@@ -248,7 +248,7 @@ version (Have_diet_ng)
 	import diet.traits;
 
 	@dietTraits
-	private struct DefaultFilters {
+	struct DefaultFilters {
```

Comment:
This should be renamed to `DefaultDietFilters`, so that it makes sense in the public context.